src/main/resources/db/migration/V022__add_in_person_data.sql

```diff
@@ -0,0 +1,10 @@
+
+COPY public.search_inperson (id, is_in_person, court_id, access_scheme) FROM stdin;
+9	f	1479943	f
+10	f	1480140	f
+11	f	1480143	f
+12	f	1480144	f
+13	f	1480146	f
+14	f	1480147	f
+15	f	1480148	f
```

Comment:
I think if you include the id it messes up the sequence  and when you try and add a row it will say somethign like 'id 9 is already used'